cosmos/operators/local.py

```diff
@@ -574,7 +574,9 @@ def upload_to_cloud_storage(self, project_dir: str) -> None:
         )
 
         for filename in self.required_files:
-            logger.info("Uploading %s to %s", filename, f"s3://{self.bucket_name}/{filename}")
+            key = f"{self.folder_dir}/{filename}" if self.folder_dir else filename
+            s3_path = f"s3://{self.bucket_name}/{key}"
+            logger.info("Uploading %s to %s", filename, s3_path)
 
             key = f"{self.folder_dir}/{filename}" if self.folder_dir else filename
```

Comment:
hey sorry if my first comment wasn't clear, `key` is already defined below on line 581 here which can be used, since it's now defined twice:
```suggestion
            key = f"{self.folder_dir}/{filename}" if self.folder_dir else filename
            s3_path = f"s3://{self.bucket_name}/{key}"
            logger.info("Uploading %s to %s", filename, s3_path)
```

Reply:
didn`t see it, my bad